tests/framework-tests/preferences/js/LetterSpaceEnactorTests.js

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2017 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid, jqUnit */
+
+(function ($) {
+    "use strict";
+
+    fluid.registerNamespace("fluid.tests");
+
+    /*******************************************************************************
+     * Unit tests for fluid.prefs.enactor.letterSpace
+     *******************************************************************************/
+
+    fluid.defaults("fluid.tests.prefs.enactor.letterSpaceEnactor", {
+        gradeNames: ["fluid.prefs.enactor.letterSpace"],
+        model: {
+            value: 1
+        },
+        fontSizeMap: fluid.tests.enactors.utils.fontSizeMap
+    });
+
+    fluid.defaults("fluid.tests.letterSpaceTests", {
+        gradeNames: ["fluid.test.testEnvironment"],
+        components: {
+            letterSpace: {
+                type: "fluid.tests.prefs.enactor.letterSpaceEnactor",
+                container: ".flc-letterSpace",
+                createOnEvent: "{letterSpaceTester}.events.onTestCaseStart"
+            },
+            letterSpaceTester: {
+                type: "fluid.tests.letterSpaceTester"
+            }
+        }
+    });
+
+    fluid.defaults("fluid.tests.letterSpaceTester", {
```

Comment:
This test should also test variants of the combination of `letter-spacing` being set or not set on the root html container and the letter space container.